tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -59,50 +59,49 @@ export class ScalarCardDataTable {
   @Output() sortDataBy = new EventEmitter<SortingInfo>();
   @Output() orderColumns = new EventEmitter<ColumnHeader[]>();
 
-  getMinValueInRange(
+  getMinPointInRange(
     points: ScalarCardPoint[],
     startPointIndex: number,
     endPointIndex: number,
     smoothed: boolean = false
   ): ScalarCardPoint {
-    let minValue = this.maybeSmoothedValue(points[startPointIndex], smoothed);
+    let minValue = points[startPointIndex].value;
     let minValuePoint = points[startPointIndex];
     for (let i = startPointIndex; i <= endPointIndex; i++) {
-      if (minValue > this.maybeSmoothedValue(points[i], smoothed)) {
-        minValue = this.maybeSmoothedValue(points[i], smoothed);
+      if (minValue > points[i].value) {
+        minValue = points[i].value;
         minValuePoint = points[i];
       }
     }
     return minValuePoint;
   }
 
-  getMaxValueInRange(
+  getMaxPointInRange(
     points: ScalarCardPoint[],
     startPointIndex: number,
-    endPointIndex: number,
-    smoothed: boolean = false
+    endPointIndex: number
   ): ScalarCardPoint {
-    let maxValue = this.maybeSmoothedValue(points[startPointIndex], smoothed);
+    let maxValue = points[startPointIndex].value;
     let maxPoint = points[startPointIndex];
     for (let i = startPointIndex; i <= endPointIndex; i++) {
-      if (maxValue < this.maybeSmoothedValue(points[i], smoothed)) {
-        maxValue = this.maybeSmoothedValue(points[i], smoothed);
+      if (maxValue < points[i].value) {
+        maxValue = points[i].value;
         maxPoint = points[i];
       }
     }
     return maxPoint;
```

Comment:
A little inconsistency: What you call "maxPoint" here you call "minValuePoint" in getMinPointInRange(). You could consier using similar terminology.